service/telemetry/internal/migration/v0.3.0.go

```diff
@@ -57,10 +59,10 @@ func (c *TracesConfigV030) Unmarshal(conf *confmap.Conf) error {
 
 type MetricsConfigV030 struct {
 	// Level is the level of telemetry metrics, the possible values are:
-	//  - "none" indicates that no telemetry data should be collected;
-	//  - "basic" is the recommended and covers the basics of the service telemetry.
-	//  - "normal" adds some other indicators on top of basic.
-	//  - "detailed" adds dimensions and views to the previous levels.
+	//  - "None" indicates that no telemetry data should be collected.
+	//  - "Basic" is the recommended and covers the basics of the service telemetry.
+	//  - "Normal" adds some other indicators on top of basic.
+	//  - "Detailed" adds dimensions and views to the previous levels.
```

Comment:
Don't know if this means any documentation/examples should also be updated or not.

Reply:
I can update documentation and examples. It technically doesn't make a difference, but I figure the closer we are to the defined constant, the less likely we are to run into problems.

Reply:
I looked into it a bit more, I think updating documentation examples may be a bit more disconcerting to users than helpful. With the widespread use of these constants in other components (such as the debug exporter), I think it's not worth it. There are also [tests to ensure case doesn't impact results](https://github.com/open-telemetry/opentelemetry-collector/blob/ea38dc4a251a1a0409277601f52b4465e43b2eee/config/configtelemetry/configtelemetry_test.go#L19), so it should be safe to keep as-is for now.

I can revert this change or leave as-is, let me know if you have a preference.